docs/meshstack.monitoring.md

```diff
@@ -24,6 +24,63 @@ These email inboxes be configured in the [meshStack configuration model](meshsta
 }
 ```
 
+## Health endpoint
```

Comment:
It is not a single health endpoint right? We have multiples

Reply:
Yes. We've the health endpoint (status, db connection status), metrics (uptime) and version.